.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     docker:
       - image: circleci/node:10
       - image: trufflesuite/ganache-cli:v6.9.1
-        command: ganache-cli -k istanbul
+        command: ganache-cli -k istanbul -e 10000000000
```

Comment:
Is that change intentional?

Reply:
Yea default balance is 100 eth. This may be overkill but allows the tests to fund more for a pair, given a more realistic exchange rate & slippage.